cmd/repo-updater/repos/github.go

```diff
@@ -400,6 +404,11 @@ func (s *GithubSource) listRepositoryQuery(ctx context.Context, query string) (m
 		return nil, nil
 	}
 
+	// Special-casing for `org:<org-name>`
```

Comment:
I'd change the comment to actually describe why it's special-casing the query, not that it's special-cased :)

Something like this:

```suggestion
	// Use the optimized org-listing that's not bound by search API limits if query is a single `org:<org-name>`
```